### PR TITLE
Add build day in the banner

### PR DESCRIPTION
**Pull request by MarcoRossignoli**
image

It's useful have the build day to speedup the eventual investigation

### CODE REVIEW

File: src/Platform/Microsoft.Testing.Platform/OutputDevice/ConsoleDisplayService.cs [hunk 1]
```diff
@@ -144,7 +145,17 @@ public async Task DisplayBannerAsync()
...
                     {
-                        stringBuilder.AppendLine(CultureInfo.InvariantCulture, $"Version: {version.InformationalVersion}");
+                        stringBuilder.Append(CultureInfo.InvariantCulture, $"Version: {version.InformationalVersion}");
+
+                        var buildTime = Assembly.GetExecutingAssembly().GetCustomAttributes(typeof(AssemblyMetadataAttribute))
+                        .OfType<AssemblyMetadataAttribute>().FirstOrDefault(x => x.Key == BUILDTIME_ATTRIBUTE_NAME);
+                        if (buildTime is not null)
+                        {
+                            DateTime buildDateTime = DateTime.FromOADate(double.Parse(buildTime.Value!, CultureInfo.InvariantCulture));
+                            stringBuilder.Append(CultureInfo.InvariantCulture, $" ({buildDateTime.ToString("MM/dd/yy", CultureInfo.InvariantCulture)})");
```
Comment (Evangelink) [inline src/Platform/Microsoft.Testing.Platform/OutputDevice/ConsoleDisplayService.cs]: Shall we use locale preference for the date display? If not, I would maybe go for the YYYY/MM/dd format as it's less confusing globally.

File: src/Platform/Microsoft.Testing.Platform/OutputDevice/ConsoleDisplayService.cs [hunk 2]
```diff
@@ -144,7 +145,17 @@ public async Task DisplayBannerAsync()
                     var version = (AssemblyInformationalVersionAttribute?)Assembly.GetExecutingAssembly().GetCustomAttribute(typeof(AssemblyInformationalVersionAttribute));
                     if (version is not null)
                     {
-                        stringBuilder.AppendLine(CultureInfo.InvariantCulture, $"Version: {version.InformationalVersion}");
+                        stringBuilder.Append(CultureInfo.InvariantCulture, $"Version: {version.InformationalVersion}");
+
+                        var buildTime = Assembly.GetExecutingAssembly().GetCustomAttributes(typeof(AssemblyMetadataAttribute))
+                        .OfType<AssemblyMetadataAttribute>().FirstOrDefault(x => x.Key == BUILDTIME_ATTRIBUTE_NAME);
```
Comment (Evangelink) [inline src/Platform/Microsoft.Testing.Platform/OutputDevice/ConsoleDisplayService.cs]:
```suggestion
                        var buildTime = Assembly.GetExecutingAssembly()
                            .GetCustomAttributes(typeof(AssemblyMetadataAttribute))
                            .OfType<AssemblyMetadataAttribute>()
                            .FirstOrDefault(x => x.Key == BUILDTIME_ATTRIBUTE_NAME);
```

File: src/Platform/Microsoft.Testing.Platform/OutputDevice/ConsoleDisplayService.cs [hunk 3]
```diff
@@ -144,7 +145,17 @@ public async Task DisplayBannerAsync()
...
+                        if (buildTime is not null)
+                        {
+                            DateTime buildDateTime = DateTime.FromOADate(double.Parse(buildTime.Value!, CultureInfo.InvariantCulture));
```
Comment (Evangelink) [inline src/Platform/Microsoft.Testing.Platform/OutputDevice/ConsoleDisplayService.cs]: Might be worth doing a null check and `TryParse`, isn't it?

Comment (MarcoRossignoli) [reply src/Platform/Microsoft.Testing.Platform/OutputDevice/ConsoleDisplayService.cs]: it cannot fail it's a `UtcNow.ToOADate()` or?

File: src/Platform/Microsoft.Testing.Platform/OutputDevice/ConsoleDisplayService.cs (hunk 1 above)
Comment (MarcoRossignoli) [reply src/Platform/Microsoft.Testing.Platform/OutputDevice/ConsoleDisplayService.cs]: I think that (UTC  YYYY/MM/dd) is good

Comment (MarcoRossignoli) [reply src/Platform/Microsoft.Testing.Platform/OutputDevice/ConsoleDisplayService.cs]: image

wdyt? too much?

Comment (Evangelink) [reply src/Platform/Microsoft.Testing.Platform/OutputDevice/ConsoleDisplayService.cs]: No no it's good and makes it explicit

**Review by Evangelink — DISMISSED** [review]

File: src/Platform/Microsoft.Testing.Platform/OutputDevice/ConsoleDisplayService.cs (hunk 3 above)
Comment (MarcoRossignoli) [reply src/Platform/Microsoft.Testing.Platform/OutputDevice/ConsoleDisplayService.cs]: Try[ing] it

**Pull request merged**